README.md

```diff
@@ -37,31 +41,24 @@ string in an Event definition.
 
 ### Installation
 
-This is an extension, so before you install Buffalo-Azure, make sure you've already [installed Buffalo](https://gobuffalo.io/en/docs/installation).
+This is an extension, so before you install Buffalo-Azure, make sure you've already 
+[installed Buffalo](https://gobuffalo.io/en/docs/installation).
 
 #### Build from Source
 
-##### option 1 - curl
+##### go get
 
-> [`dep`](https://github.com/golang/dep) is a pre-requisite for these installation instructions.
-
-If you're using MacOS or Linux, you can get started by running our installation script:
-
-``` bash
-curl https://raw.githubusercontent.com/Azure/buffalo-azure/master/install.sh | sh
-```
-
-##### option 2 - go get  
-
-The most global, albeit least stable, way to get started is to use `go get`.
+The way to get started is to use `go get`. This project support Go modules when using Go 1.11 and above.
+For more information see [Go modules wiki](https://github.com/golang/go/wiki/Modules)
```

Comment:
[nit] grammar.

I'd update this to:
```
Use `go get` to acquire and install from source. Versions of the project after {insert the version this will be} use Go modules exclusively, which means you'll need Go 1.11 or later.

For more information on modules, see the [Go modules wiki](https://github.com/golang/go/wiki/Modules).
```

Reply:
done